web_src/js/components/RepoActionView.vue

```diff
@@ -185,11 +186,72 @@ export default defineComponent({
     document.body.addEventListener('click', this.closeDropdown);
     this.hashChangeListener();
     window.addEventListener('hashchange', this.hashChangeListener);
+
+    // === Auto Expand + Auto Scroll Fix (for Issue #35570) ===
+    // Ensure Vue has updated DOM for steps after initial load
+    await nextTick();
+
+    // Set up observer on the steps container (safer and more efficient than document.body)
+    const stepsContainer = (this.$refs.steps as HTMLElement);
+    if (stepsContainer && typeof MutationObserver !== 'undefined') {
```

Comment:
```suggestion
    if (stepsContainer && 'MutationObserver' in window) {
```

matches other `MutationObserver` usages.